Source/4_0/CentralManager.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import RxSwift
+import CoreBluetooth
+
+public final class CentralManager: CentralManagerType {
+
+    public let cbCentralManager: CBCentralManager
+
+    init(cbCentralManager: CBCentralManager) {
+        self.cbCentralManager = cbCentralManager
+    }
+
+    /// Observable which emits state changes of central manager after subscriptions
+    var rx_didUpdateState: Observable<BluetoothState> {
+        return .empty()
+    }
```

Comment:
Ok, so I think that these should be written like that, or maybe `_rxDidUpdateState`? Also, I have problem when writing extension to reactive: ```extension Reactive where Base: ManagerType {
    public var mm: Observable<BluetoothState> {
        return .deferred {
            return self.base.rx_didUpdateState.startWith(self.base.state)
        }
    }
}```
It doesn't compile, unfortunately.